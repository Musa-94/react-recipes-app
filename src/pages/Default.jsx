import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';

class Default extends PureComponent {
    render() {
        return (
            <div>
                <h1> Hello Error Page </h1>
            </div>
        );
    }
}

Default.propTypes = {};

export default Default;
