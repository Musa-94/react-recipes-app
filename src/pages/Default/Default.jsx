import React from 'react';
import Header from '../../components/Header/Header.jsx';

const Default = () => {
    return(
        <Header
            title={'404'}
            styleClass={'default-hero'}
        >
            <h2 children={'You are in the wrong place'}
                className={'text-light text-uppercase'}
            />
            <a href={'#/'}
               children={'Return Home'}
               className={'text-uppercase btn btn-secondary btn-lg mt-3'}
            />
        </Header>
    );
}

export default React.memo(Default);
