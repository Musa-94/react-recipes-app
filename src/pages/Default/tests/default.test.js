import React from 'react';
import Default from '../Default';

describe('Default snapshot', () => {
    it('should correct renderer', () => {
        const component = renderer(<Default/>);

        expect(component).toMatchSnapshot();
    });
});