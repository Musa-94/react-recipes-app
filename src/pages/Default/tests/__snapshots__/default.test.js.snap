// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default snapshot should correct renderer 1`] = `
<header>
  <div
    className="container-fluid"
  >
    <div
      className="row align-items-center default-hero"
    >
      <div
        className="col text-center text-slanted"
      >
        <h1
          className="text-light text-uppercase display-3 letter-spacing text-slander"
        >
          404
        </h1>
        <h2
          className="text-light text-uppercase"
        >
          You are in the wrong place
        </h2>
        <a
          className="text-uppercase btn btn-secondary btn-lg mt-3"
          href="#/"
        >
          Return Home
        </a>
      </div>
    </div>
  </div>
</header>
`;
